scripts/ablexicon

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+#
+# ablexicon - Compare an input list of words against a dictionary and
+# optional lexicon.  If any words are in neither the dictionary nor the
+# lexicon, log them to stdout.
+#
+set -e
+set -f
+
+function usage () {
+    echo "Find occurrences of non-dictionary/lexicon words"
+    echo ""
+    echo "Usage:"
+    echo " ${0##*/} [options]"
+    echo ""
+    echo "Options:"
+    echo " -f, --file         source text (defaults to stdin)"
+    echo " -l, --lexicon      lexicon file (one word per line)"
+    echo " -h, --help         display this help"
+    exit 1
+}
+
+#
+# Verify that required commands are present
+#
+REQUIRED=( "spell" "getopt" )
+for i in "${REQUIRED[@]}"
+do
+    command -v $i"" >/dev/null
+    if [ $? -ne "0" ]
+    then
+        echo "'"$i"' must be installed, exiting...">&2
+        exit 1
+    fi
+done
+
+GETOPT_OUT=`getopt -o hf:l: --long help,file:,lexicon: -n "${0##*/}" -- "$@"`
+if [ $? != 0 ]
+then
+    echo "Exiting..." >&2
+    exit 1
+fi
+
+eval set -- "$GETOPT_OUT"
+
+INFILE=/dev/stdin
```

Comment:
Is there a way to make this more cross-platform like `/dev/fd/0`? It would be valuable to us to be able to run this in our development environment without relying on Travis/switching to Ubuntu

Reply:
Changed to `/dev/fd/0` throughout.